newIDE/app/src/MainFrame/EditorContainers/HomePage/BuildSection/ProjectFileListItem.js

```diff
@@ -311,15 +313,13 @@ export const ProjectFileListItem = ({
     }
 
     const gameId = file.fileMetadata.gameId;
-    if (gameId) {
+    if (gameId && onManageGame && canManageGame) {
       actions = actions.concat([
         { type: 'separator' },
         {
           label: i18n._(t`Manage game`),
-          click: () =>
-            navigateToRoute('games-dashboard', {
-              'game-id': gameId,
-            }),
+          click: () => onManageGame({ gameId }),
+          enabled: canManageGame({ gameId }),
```

Comment:
should the whole concat not happen if `!canManageGame( {gameId})` ? It would avoid showing an item that's not clickable?

Reply:
I usually prefer disabling items rather than having users think "there's a bug, this item should be here but it's not"